package com.taskManager.exception;

public class PersonNotFoundException extends RuntimeException{

    public PersonNotFoundException() {
        super("Usuario n√£o encontrado!!");
    }
}
